### DISCUSSION TITLE
zotero 7装了茉莉花插件，无法更新非官方中文转换器

### DISCUSSION THREAD

**Discussion by aaalu26** [l0o0/jasminum#290] [Q&A] [answered]
如图，这个更新的地方一直在转圈圈，加载不出来
![image](https://github.com/user-attachments/assets/91230d73-f7e3-499d-904f-b0f0c2bce8c1)

**Comment by jiaojiaodubai** [comment, marked as answer]
请到插件商店下载安装最新版

**Reply by aaalu26** [reply]
解决了，感谢！！
